### PR TITLE
[Facilities] Building page updates

### LINKED ISSUE #7408: Facilities feedback
- Named building no longer populated on buildings
- See if we can shorten up the length of the building pages
- Disclaimer text - two sentences - Wendy to send what is currently still under review.
- ~let them know when we deploy searchable/filterable list of building coordinators so they can share with building coordinator manager.~ - done

### PR DESCRIPTION
**Pull request by bspeare**
Resolves https://github.com/uiowa/uiowa/issues/7408

# How to test

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local fm-buildings && ddev drush @facilities.local uli
```

- Code review
- Review layouts with different amount of content
- Confirmed Named building field is displaying on https://facilities.uiowa.ddev.site/building/0008

- https://facilities.uiowa.ddev.site/building/0001
- https://facilities.uiowa.ddev.site/building/0007
- https://facilities.uiowa.ddev.site/building/0008
- https://facilities.uiowa.ddev.site/building/0009
- https://facilities.uiowa.ddev.site/building/0018
- https://facilities.uiowa.ddev.site/building/0025
- https://facilities.uiowa.ddev.site/building/0033

### CODE REVIEW

**Comment by joewhitsitt** [comment]
pulling to test

File: docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module [hunk 1]
```diff
@@ -239,9 +239,33 @@ function facilities_core_preprocess_field(&$variables) {
...
       }
       break;
 
+    case 'field_building_lactation_rooms':
+    case 'field_building_rr_single_neutral':
+    case 'field_building_rr_single_men':
+    case 'field_building_rr_single_women':
+    case 'field_building_rr_multi_men':
+    case 'field_building_rr_multi_women':
+      // Wrap the field output in a <details> tag.
```
Comment (joewhitsitt) [inline docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module]: This is fine, but seems like a fairly simple render array. https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21Element%21Details.php/class/Details/10

File: docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module [hunk 2]
```diff
@@ -267,17 +291,14 @@ function facilities_core_preprocess_field(&$variables) {
...
-      ]);
+    case 'field_building_area':
+    case 'field_building_named_building':
+    case 'field_building_ownership':
+      uiowa_core_set_attributes($variables, 'padding--inline-end--md');
       break;
 
-    case 'field_building_lactation_rooms':
+    case 'field_building_hours':
+      $variables['items'][0]['content']['#suffix'] = '<br /><details><summary>Disclaimer Statement</summary><small>Public operating hours for campus buildings are displayed, excluding holidays and special events. These hours do not represent office business hours. Access to buildings during closed hours is permitted via the Iowa One Card for approved faculty, staff, and students. For Hospital 24/7 entrances, visitor guidelines, and information, please visit the <a href="https://uihc.org/visitor-guidelines-ui-health-care">UI Health Care website</a>.</small></summary>';
```
Comment (joewhitsitt) [inline docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module]: I think it autoclosed on render, but i feel like this is missing a `</details>`

**Comment by joewhitsitt** [comment]
Nitpicking here, but is there an easy way to tie this `<hr>` to the existence of content below so it doesn't show in these instances.

Screenshot 2024-03-01 at 9 54 57 AM

**Review by joewhitsitt — APPROVED** [review]
Overall it is looking good and delivers on the "tightening up" deliverable.

File: docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module (hunk 1 above)
Comment (bspeare) [reply docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module]: Great suggestion! Should be resolved in https://github.com/uiowa/uiowa/pull/7409/commits/b9346a14e195025d21753013314857334f66c61e.

File: docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module (hunk 2 above)
Comment (bspeare) [reply docroot/sites/facilities.uiowa.edu/modules/facilities_core/facilities_core.module]: Thanks! This should also be resolved in https://github.com/uiowa/uiowa/pull/7409/commits/b9346a14e195025d21753013314857334f66c61e.

**Pull request merged**